### DISCUSSION TITLE
How to disable 'Playing Logicism' while my status is idle?

### DISCUSSION THREAD

**Discussion by SANTERCOREJZ** [LogicismDev/LogRPC#3] [Q&A]
I don't like that thing, is it possible to disable that?

**Comment by LogicismDev** [comment]
Can you clarify more on the question?

Basically if you right click on the LogRPC Icon and click on Discord - yourusername, it should toggle it off.

If you meant that if you want to have it be disabled after a period of time, the feature isn't implemented. But I can make it a feature if you wish.
